students/mattclau/lesson02/grid.py

```diff
@@ -0,0 +1,27 @@
+corner = '+'
+horz = '-'
+vert = '|'
+space = ' '
+
+def printHoriz(gridSize, sideLen):
```

Comment:
Per PEP8 function names and variables should be snake_case.  def print_horiz(grid_size, side_len):